code/game/machinery/doors/airlock.dm

```diff
@@ -72,6 +72,7 @@
 	var/hasShocked = 0 //Prevents multiple shocks from happening
 	autoclose = 1
 	var/obj/item/device/doorCharge/charge = null //If applied, causes an explosion upon opening the door
+	var/obj/item/weapon/note //Any papers pinned to the airlock
```

Comment:
Don't worry about it, man. It's not hard work, just tedious, and I wasn't really sure if it was worth it considering I've seen people complain about resource bloat recently. Besides, it's not your fault that Remie and I had a disagreement.